scripts/build-package.js

```diff
@@ -0,0 +1,36 @@
+const glob = require('glob');
+const path = require('path');
+const npmRun = require('npm-run');
+const yargs = require('yargs');
+
+/**
+ * Runs the build-babel or build-webpack npm target for the specified package.
+ * @param packageName Name of the package as listed in ./packages/
+ */
+function runBuildTask(packageName) {
+  const npmTask = packageName !== 'react-component-playground' ? 'build-babel' :
+      'build-webpack';
+  const stdout = npmRun.execSync(`PACKAGE=${packageName} npm run ${npmTask}`, {
+    cwd: __dirname,
+  });
+
+  /* eslint-disable no-console */
+  console.log(stdout.toString('utf-8'));
+}
+
+glob('./packages/*react-*', null, (err, files) => {
```

Comment:
Shouldn't this be `./packages/react-*`to only match packages that *start* with `react`?